### ISSUE TITLE
[cryptocoins@pbojan] Icons stretched on HighDPI

### ISSUE THREAD

**Issue by Redsandro** [linuxmint/cinnamon-spices-desklets#464]
```
 * Desklet version 1.2
 * Cinnamon version 4.2.4
 * Distribution - Mint 19.2
 ```

**Issue**

Icons are stretched on a scaled ui.

![image](https://user-images.githubusercontent.com/1702193/69454873-7beacf00-0d67-11ea-902b-0df2fdf899c7.png)

Tried to write PR but I can't figure out what's causing this.

**Comment by NikoKrause** [comment]
ping author @pbojan

**Comment by pbojan** [comment]
This should be fixed with v1.5 (https://github.com/linuxmint/cinnamon-spices-desklets/pull/638). As I don't really have a proper HiDPI monitor would be nice if @Redsandro can verify this after the version is live?

**Comment by Redsandro** [comment]
Works fine now. Screenshot added next to panel for measure.

![image](https://user-images.githubusercontent.com/1702193/113637471-f4101700-9674-11eb-8a92-f50832d51e05.png)
